pom.xml

```diff
@@ -1366,10 +1366,12 @@ under the License.
 								<configuration>
 									<rules>
 										<requireMavenVersion>
-											<version>3.8.6</version>
+											<!-- Enforce the Maven version that's currently used within Flink's CI pipelines. -->
+											<version>[3.8.6]</version>
 										</requireMavenVersion>
 										<requireJavaVersion>
-											<version>1.8.0</version>
+											<!-- Enforce the default Java target version for releasing build artifacts. -->
+											<version>[1.8.0,1.8.1)</version>
```

Comment:
I wonder whether we should a comment like 
`Please keep it in sync with https://cwiki.apache.org/confluence/display/FLINK/Creating+a+Flink+Release`

Reply:
That sounds reasonable. I'm just wondering whether, on the contrary, we should remove the information from the release documentation since it's enforced now through code, anyway. This way we would reduce the effort around maintaining documentation. :thinking: 

But it doesn't hurt that much, I guess. So, I go with your proposal.